portingdb/templates/namingpolicy.html

```diff
@@ -0,0 +1,55 @@
+{% extends "_base.html" %}
+
+{% block titlecontent -%}
+    Misnamed Packages – {{ super() }}
+{%- endblock titlecontent %}
+
+{% block bodycontent %}
+
+<div class="container">
+    <div class="col-md-8">
+        <h1>Naming Policy Violations</h1>
+    </div>
+    <div class="col-md-12">
+        {{ progress_summary(progress, total_packages) }}
+    </div>
+    <div class="col-md-12">
+        <p>
+            This is a dashboard to track packages which do not comply with 
+            <a
+               href="https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Python_source_package_naming" 
+               target="_blank">Python package naming guidelines</a>.
+            These packages are to follow the new guidelines before 2020 / Fedora 29,
```

Comment:
As we do not know the release number yet, I just removed it and only left the year (in 72ed03b)